### PR TITLE
Style changes

### PR DESCRIPTION
**Pull request by ameliaeiger**
Added styling, small refactor to structure, conditionally rendering components, fixed scroll bar so it is horizontal.

### CODE REVIEW

File: Views/Dashboard.js [hunk 1]
```diff
@@ -9,56 +9,102 @@ import { Button } from 'react-native-paper'
...
+import { useFonts } from "expo-font"
 import Results from "../components/Results"
 import Data from "./data.js"
+import * as Haptics from 'expo-haptics';
+
+
+export default function Dashboard({ navigation, handleChange, allImages, images, dimensions }) {
+  const [hasImages, setHasImages] = useState("")
+
+  useEffect(() => {
```
Comment (BlueNealis) [inline Views/Dashboard.js:21]: Using that useEffect, like a bosss

File: Views/Welcome.js [hunk 2]
```diff
@@ -6,32 +6,30 @@ import { useState, useEffect } from "react"
...
-    const [height, setHeight] = useState(screenDimensions.height)
-    const [width, setWidth] = useState(screenDimensions.width)
-
     const [loaded] = useFonts({
         Poppins: require('../assets/poppinsLight.ttf'),
       })
       if (!loaded) {
-        return null
+        return <Text>loading</Text>
       }
```
Comment (BlueNealis) [inline Views/Welcome.js:16]: Nice thanks for adding a loading text

File: Views/Dashboard.js [hunk 3]
```diff
@@ -9,56 +9,102 @@ import { Button } from 'react-native-paper'
...
-              </View>
-            </View>
+    <View style={{
+      width: dimensions.width,
+      maxHeight: dimensions.height,
+      }}>
+        {!hasImages ? displayText() : displayImages()}
+      <View style={styles.buttonContainer}>
+        <View style={{flexDirection:"row"}}>
+          <PickImage handleChange={handleChange}/>
```
Comment (BlueNealis) [inline Views/Dashboard.js:62]: Nice itll be in a row now yay

**Review by BlueNealis — APPROVED** [review]
Excited for these changes yay!

**Review by Neil-B-Linton — APPROVED** [review]
Wicked cool styling!

**Review by laytonmaes — APPROVED** [review]

**Pull request merged**